versioneer.py

```diff
@@ -277,17 +277,19 @@
 """
 
 from __future__ import print_function
-try:
-    import configparser
-except ImportError:
-    import ConfigParser as configparser
+
 import errno
 import json
 import os
 import re
 import subprocess
 import sys
 
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+
```

Comment:
Why?

Reply:
this was auto-formatted by isort, will undo